sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -1,15 +1,16 @@
-# Azure Text Analytics client library for Java
-Text Analytics is a cloud-based service that provides advanced natural language processing over raw text, 
-and includes the main features below:
+# Azure Language service client library for Java
```

Comment:
I think this should stay "Text Analytics" since that's still the name of the client library, it just uses the language service